simde/arm/neon/cvt.h

```diff
@@ -1182,8 +1190,6 @@ simde_vcvtas_s32_f32(simde_float32 a) {
     } else if (HEDLEY_UNLIKELY(simde_math_isnanf(a))) {
       return 0;
     } else {
-      // Round to Nearest with Ties to Away (a.k.a Rounding away from zero) rounding mode.
-      // For example, 23.2 gets rounded to 24, and −23.2 gets rounded to −24.
```

Comment:
Is this not true anymore?

Reply:
The definition of the rounding mode is correct, but it only occurs when the number is at the midpoint. Otherwise, it rounds to the nearest integer. The example below is incorrect. For example, `23.2` gets rounded to `23`, and `23.5` gets rounded to `24`.